lib/govuk_component/button_helpers.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module GovukComponent
+  module ButtonHelpers
+    def govuk_button(**tag_options, &block)
+      tag_options = prepend_classes('govuk-button', tag_options)
+      tag_options[:data] = { module: 'govuk-button', 'prevent-double-click': 'true' }
+      is_disabled(tag_options)
+      content = capture(&block)
+
+      tag.button(content, tag_options)
+    end
+
+    def govuk_button_secondary(**tag_options, &block)
+      tag_options = prepend_classes('govuk-button--secondary', tag_options)
+
+      govuk_button(tag_options, &block)
+    end
+
+    def govuk_button_warning(**tag_options, &block)
+      tag_options = prepend_classes('govuk-button--warning', tag_options)
+
+      govuk_button(tag_options, &block)
+    end
+
+    def govuk_button_start(**tag_options, &block)
+      tag_options = prepend_classes('govuk-button govuk-button--start', tag_options)
+      tag_options[:data] = { module: 'govuk-button' }
+      tag_options[:draggable] = 'false'
+      tag_options[:role] = 'button'
+      content = capture(&block)
+
+      svg_path = tag.path(fill: 'currentColor', d: 'M0 0h13l20 20-20 20H0l20-20z')
+      svg_options = {}
+      svg_options[:class] = 'govuk-button__start-icon'
+      svg_options[:xmlns] = 'http://www.w3.org/2000/svg'
+      svg_options[:height] = '19'
+      svg_options[:width] = '17.5'
+      svg_options[:viewBox] = '0 0 33 40'
+      svg_options[:aria] = { hidden: 'true' }
+      svg_options[:focusable] = 'false'
```

Comment:
could refactor this out to a `svg_options` method or constant since they are hardcoded